great_expectations/metrics/column_map/between.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional
+
+from great_expectations.core.types import Comparable
+from great_expectations.metrics import Metric
+from great_expectations.metrics.domain import ColumnMap
+
+
+class Between(Metric, ColumnMap):
+    min_value: Optional[Comparable] = None
+    max_value: Optional[Comparable] = None
+    strict_min: bool = False
+    strict_max: bool = False
```

Comment:
We still need to add a docstring and tests for this. I only put it in the PR as a demonstration of how it will work.